test/e2e/rukpakctl_test.go

```diff
@@ -0,0 +1,297 @@
+				Eventually(func() (string, error) {
+					if err := c.Get(ctx, client.ObjectKeyFromObject(bundle), bundle); err != nil {
+						return "", err
+					}
+					return bundle.Status.Phase, nil
+				}).Should(Equal(rukpakv1alpha1.PhaseUnpacked))
+				Eventually(func() (*metav1.Condition, error) {
+					if err := c.Get(ctx, client.ObjectKeyFromObject(bundle), bundle); err != nil {
+						return nil, err
+					}
+					return meta.FindStatusCondition(bundle.Status.Conditions, rukpakv1alpha1.TypeUnpacked), nil
+				}).Should(And(
+					Not(BeNil()),
+					WithTransform(func(c *metav1.Condition) string { return c.Type }, Equal(rukpakv1alpha1.TypeUnpacked)),
+					WithTransform(func(c *metav1.Condition) metav1.ConditionStatus { return c.Status }, Equal(metav1.ConditionTrue)),
+					WithTransform(func(c *metav1.Condition) string { return c.Reason }, Equal(rukpakv1alpha1.ReasonUnpackSuccessful)),
+				))
+			})
+			bundledeployment = &rukpakv1alpha1.BundleDeployment{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: bundledeploymentname,
+				},
+			}
+			By("eventually reporting Unpacked", func() {
+				Eventually(func() (*metav1.Condition, error) {
+					if err := c.Get(ctx, client.ObjectKeyFromObject(bundledeployment), bundledeployment); err != nil {
+						return nil, err
+					}
+					return meta.FindStatusCondition(bundledeployment.Status.Conditions, rukpakv1alpha1.TypeHasValidBundle), nil
+				}).Should(And(
+					Not(BeNil()),
+					WithTransform(func(c *metav1.Condition) string { return c.Type }, Equal(rukpakv1alpha1.TypeHasValidBundle)),
+					WithTransform(func(c *metav1.Condition) metav1.ConditionStatus { return c.Status }, Equal(metav1.ConditionTrue)),
+					WithTransform(func(c *metav1.Condition) string { return c.Reason }, Equal(rukpakv1alpha1.ReasonUnpackSuccessful)),
+				))
+			})
+			By("eventually reporting Installed", func() {
+				Eventually(func() (*metav1.Condition, error) {
+					if err := c.Get(ctx, client.ObjectKeyFromObject(bundledeployment), bundledeployment); err != nil {
+						return nil, err
+					}
+					return meta.FindStatusCondition(bundledeployment.Status.Conditions, rukpakv1alpha1.TypeInstalled), nil
+				}).Should(And(
+					Not(BeNil()),
+					WithTransform(func(c *metav1.Condition) string { return c.Type }, Equal(rukpakv1alpha1.TypeInstalled)),
+					WithTransform(func(c *metav1.Condition) metav1.ConditionStatus { return c.Status }, Equal(metav1.ConditionTrue)),
+					WithTransform(func(c *metav1.Condition) string { return c.Reason }, Equal(rukpakv1alpha1.ReasonInstallationSucceeded)),
+				))
+			})
+		})
+	})
+	When("run executed with an invalid manifest directory", func() {
+		var (
+			message string
+		)
+		BeforeEach(func() {
+			out, err := exec.Command("sh", "-c", "../../bin/rukpakctl run test ../../testdata/bundles/plain-v0/notvalid").CombinedOutput()
+			message = string(out)
+			Expect(err).NotTo(BeNil())
+		})
+		It("should fail by `no such file or directory`", func() {
```

Comment:
nit:
```suggestion
		It("should fail with a "no such file or directory" message", func() {
```

Reply:
@tylerslaton Thanks for review.  This one is waiting for #417 settled.  This will be reworked for it.

Reply:
I change this.  Thanks!